test/general/streaming.js

```diff
@@ -234,6 +234,8 @@ function tests() {
   });
 
   it('Encrypt and decrypt larger message roundtrip', async function() {
+    let aead_protectValue = openpgp.config.aead_protect;
```

Comment:
Hey, In my opinion it would be better to use consistent casing. We use camelCase as default one.